src/Cli.Tests/ModuleInitializer.cs

```diff
@@ -21,6 +21,12 @@ public static void Init()
         VerifierSettings.IgnoreMember<DataSource>(dataSource => dataSource.ConnectionString);
         // Ignore the JSON schema path as that's unimportant from a test standpoint.
         VerifierSettings.IgnoreMember<RuntimeConfig>(config => config.Schema);
+        // Ignore the databaseNameToDataSource array as its not exposed to customer.
+        VerifierSettings.IgnoreMember<RuntimeConfig>(config => config.DatasourceNameToDataSource);
```

Comment:
nit: rename to `DataSourceNameToDataSource`

Reply:
fixed.